doc/ko/uftrace-dump.md

```diff
@@ -114,7 +114,7 @@ DUMP 옵션
     기본 설정은 "simple"이며, 템플릿 파라미터와 함수 인자를 무시한다.
 
 -r *RANGE*, \--time-range=*RANGE*
-:   시간 범위 RANGE 내에 실행된 함수들만 출력한다. RANGE 는 \<시작\>~\<끝\>
+:   시간 범위 RANGE 내에 실행된 함수들만 출력한다. RANGE 는 \<시작\> ~ \<끝\>
```

Comment:
`~` shouldn't have empty space before and after because it's a syntax to use it without space.

Please remove `~` related changes from this PR.